resource_management_mobile/src/app/auth/login/login.page.scss

```diff
@@ -0,0 +1,83 @@
+
+$font-weight:600;
+$curve-size:47px;
+@mixin flexDisplay {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+}
+
+
+.login_header{
+    @include flexDisplay;
+    padding-top: 18%;
+    color: white;
+}
+.header_text{
+    font-weight:$font-weight;
+    letter-spacing: 9px;
+}
+.sub_header_text{
+    margin-top: 10px;
+}
+.login_background{
+    background: linear-gradient(180deg, #050579 0%, rgba(5, 5, 121, 0) 100%);
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+    height: calc(80% + 20px);
+    position: relative;
+}
+.login_model{
+    background-color: white;
+    position: absolute;
+    top: 50%;
+    height: 50%;
+    width: 100%;
+    border-top-left-radius: $curve-size;
+    border-top-right-radius: $curve-size;
+
+}
+
+.login_data_container{
+   @include flexDisplay;
+   padding-top: 13%;
+    // margin-top: 27%;
+}
+
+.admin_login{
+    font-weight:$font-weight;
+    color: #535395;
+    font-size: 1.5em;
+}
+.item_list{
+    width: 100%;
+    padding: 3% 9%;
+    color: black;
+    // margin-top: 8%;
+}
+
+.item_list > ion-input{
+    margin-top: 7%;
```

Comment:
Don't use % on margins,paddings and font-sizes. it's not bast practice

Reply:
changed %  to px